nf_core/modules/lint/main_nf.py

```diff
@@ -163,9 +163,9 @@ def check_process_section(self, lines):
         if re.search("bioconda::", l):
             bioconda_packages = [b for b in l.split() if "bioconda::" in b]
         if re.search("org/singularity", l):
-            singularity_tag = l.split("/")[-1].replace('"', "").replace("'", "").split("--")[-1].strip()
+            singularity_tag = l.split()[0].split("/")[-1].split("/")[-1].replace('"', "").replace("'", "").split(':')[-1]
```

Comment:
Any chance I can replace all of those splits with a regular expression? It makes me ￼a little nervous that it looks quite fragile (but probably unfounded)

What is it trying to do? A code comment with an example string would help for readability too..

Reply:
Absolutely! Sorry, I had to move on to the next thing.

Reply:
It should be getting the version and build number from the container:

```python
>>> l = "'https://depot.galaxyproject.org/singularity/fastqc:0.11.9--0' :"
>>> singularity_tag = l.split()[0].split("/")[-1].split("/")[-1].replace('"', "").replace("'", "").split(':')[-1]
>>> singularity_tag
'0.11.9--0'

>>> l = "'quay.io/biocontainers/fastqc:0.11.9--0' }\""
>>> docker_tag = l.split()[0].split("/")[-1].split("/")[-1].replace('"', "").replace("'", "").split(':')[-1]
>>> docker_tag
'0.11.9--0'